standard/tier/wikis/smash/tier_data.lua

```diff
@@ -0,0 +1,139 @@
+---
+-- @Liquipedia
+-- wiki=smash
+-- page=Module:Tier/Data
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+return {
+	tiers = {
+		{
+			value = '1',
+			sort = 'A1',
+			name = 'Premier',
+			short = 'Prem.',
+			link = 'Premier Tournaments',
+			category = 'Premier Tournaments',
+		},
+		{
+			value = '2',
+			sort = 'A2',
+			name = 'Major',
+			short = 'Maj.',
+			link = 'Major Tournaments',
+			category = 'Major Tournaments',
+		},
+		{
+			value = '3',
+			sort = 'A3',
+			name = 'National',
+			short = 'Nat.',
+			category = 'National Tournaments',
+		},
+
+		-- legacy??? to be replaced by `-1` ???
```

Comment:
Tier 4 is not legacy and is named correctly. It must not be replaced by -1

Reply:
it indicates missing tier, right?

Reply:
No, it is properly tiered, but the tournament itself is lower prestige. "No Tier" indicates it is below a National.

Reply:
okay